app/uk/gov/hmrc/cgtpropertydisposals/service/TaxEnrolmentService.scala

```diff
@@ -60,7 +60,7 @@ class TaxEnrolmentServiceImpl @Inject()(connector: TaxEnrolmentConnector) extend
         maybePostcode match {
           case Some(postcode) =>
```

Comment:
should we be sending the country code here for a non-uk address?

Reply:
I'm not sure of the requirements here. I have given postcode precedence for both scenarios. Should it be country code for Non-UK addresses?

Reply:
changed to always use country-code for non-uk addresses